install/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+sh_binary(
```

Comment:
I don't think it should be a new "install" package at the top-level of silo, it's specific to the CLI right? I'd expect it to be in /cli/install

Reply:
It should. I started this PR, before we moved the source of truth. Perhaps I will close this PR and open it in silo.

Reply:
My mistake, this isn't a PR against the silo repo